iOS_SDK/OneSignalSDK/UnitTests/UnitTests.m

```diff
@@ -2013,6 +2014,120 @@ - (void)testZSwizzling {
   
 }
 
+- (NSDictionary *)setUpWillShowInForegroundHandlerTestWithBlock:(OSNotificationWillShowInForegroundBlock)willShowInForegroundBlock withDisplayType:(OSNotificationDisplayType)displayType withNotificationOpenedBlock:(OSNotificationOpenedBlock)openedBlock withPayload: (NSDictionary *)payload {
+    
+    [UnitTestCommonMethods initOneSignalWithHandlers:willShowInForegroundBlock notificationOpenedHandler:openedBlock];
+
+    [UnitTestCommonMethods runBackgroundThreads];
+
+    [OneSignal setNotificationDisplayType:displayType];
+
+    UIApplicationOverrider.currentUIApplicationState = UIApplicationStateActive;
+
+    [UnitTestCommonMethods runBackgroundThreads];
+
+    return payload;
+}
+
+- (void)fireWillPresentNotificationInForegroundHandlerWithForegroundBlock:(OSNotificationWillShowInForegroundBlock)willShowInForegroundBlock {
+    __block var option = (UNNotificationPresentationOptions)7;
+    __block var completionCount = 0;
+    let expectation = [self expectationWithDescription:@"wait_for_timeout"];
+    expectation.expectedFulfillmentCount = 1;
+    NSDictionary *payload = @{@"aps": @{
```

Comment:
sry, can you format this dictionary? In that way is easier to read

Reply:
fixed!